modules/gapi/CMakeLists.txt

```diff
@@ -178,6 +181,15 @@ ocv_module_include_directories("${CMAKE_CURRENT_LIST_DIR}/src")
 ocv_create_module()
 
 ocv_target_link_libraries(${the_module} PRIVATE ade)
+
+if(CV_TRACE AND HAVE_ITT)
+  add_definitions(-DOPENCV_WITH_ITT=1)
+endif()
+if(ITT_INCLUDE_DIRS)
+  ocv_module_include_directories(${ITT_INCLUDE_DIRS})
+endif()
+
+ocv_target_link_libraries(${the_module} PRIVATE "${ITT_LIBRARIES}")
```

Comment:
I believe this should be done under some condition, too.

Reply:
Reworked, thanks!